DockerOrchestration/ABCIApp/Dockerfile.Dummy

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8
```

Comment:
is there really an advantage to having two dockerfiles here? everything is basically the same except the final command